src/visor/blockchain.go

```diff
@@ -151,23 +151,27 @@ func (bc *Blockchain) headSeq() int64 {
 	return bc.chain.HeadSeq()
 }
 
-func (bc *Blockchain) processBlock(b *coin.Block) error {
-	if !bc.isGenesisBlock(*b) {
-		if err := bc.verifyBlockHeader(*b); err != nil {
-			return err
+func (bc *Blockchain) processBlock(b coin.Block) (coin.Block, error) {
+	if !bc.isGenesisBlock(b) {
+		if err := bc.verifyBlockHeader(b); err != nil {
+			return coin.Block{}, err
 		}
 		txns, err := bc.processTransactions(b.Body.Transactions)
```

Comment:
`processTransactions` doesn't always return a copy of `txns`, needs to always return a copy

Reply:
oh, yes, it's a slice.